README.md

```diff
@@ -36,7 +36,9 @@ Although most of the plugin code was written when an earlier spec was current:
 It also implements the FileWriter spec :
 [http://dev.w3.org/2009/dap/file-system/file-writer.html](http://dev.w3.org/2009/dap/file-system/file-writer.html)
 
-For usage, please refer to HTML5 Rocks' excellent [FileSystem article.](http://www.html5rocks.com/en/tutorials/file/filesystem/)
+>*Note* For browser targets, some of the file system APIs are deprecated, but many of the APIs are supported in Cordova for the non-browser platforms listed in the _Supported Platforms_ list (see below).
```

Comment:
I'm concerned that many readers of this may be unfamiliar with the "browser" target platform of Cordova, and potentially confused because Cordova apps on all platforms are rendered in a web browser control. Can this be made clearer?